### ISSUE TITLE
Sign out does not expire access and refresh token

### ISSUE THREAD

**Issue by mmianl** [aws-samples/cloudfront-authorization-at-edge#289]
The [sign-out handler](https://github.com/aws-samples/cloudfront-authorization-at-edge/blob/master/src/lambda-edge/sign-out/index.ts#L71) marks the ID token as expired, but it does not do so for the access and refresh tokens. This can cause the following issue:

When a user logs in as `userA`, then signs out, the access and refresh tokens will be preserved by the user's web browser, as they are not marked as expired. See the log of the sign-out handler:

```json
TRACE	Returning response:
 
{
    "status": "307",
    "statusDescription": "Temporary Redirect",
    "headers": {
        "location": [
            {
                "key": "location",
                "value": ""
            }
        ],
        "set-cookie": [
            {
                "key": "set-cookie",
                "value": "CognitoIdentityServiceProvider.xK9nubrGTDSakK8uwSL2P6WmUd.LastAuthUser=; Path=/; Secure; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            },
            {
                "key": "set-cookie",
                "value": "CognitoIdentityServiceProvider.xK9nubrGTDSakK8uwSL2P6WmUd.userA.tokenScopesString=; Path=/; Secure; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            },
            {
                "key": "set-cookie",
                "value": "CognitoIdentityServiceProvider.xK9nubrGTDSakK8uwSL2P6WmUd.userA.userData=; Path=/; Secure; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            },
            {
                "key": "set-cookie",
                "value": "amplify-signin-with-hostedUI=; Path=/; Secure; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            },
            {
                "key": "set-cookie",
                "value": "CognitoIdentityServiceProvider.xK9nubrGTDSakK8uwSL2P6WmUd.userA.idToken=; Path=/; Secure; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            },
            {
                "key": "set-cookie",
                "value": "spa-auth-edge-nonce=; Path=/; Secure; HttpOnly; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            },
            {
                "key": "set-cookie",
                "value": "spa-auth-edge-nonce-hmac=; Path=/; Secure; HttpOnly; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            },
            {
                "key": "set-cookie",
                "value": "spa-auth-edge-pkce=; Path=/; Secure; HttpOnly; SameSite=Lax; Expires=Thu, 01 Jan 1970 00:00:00 GMT"
            }
        ],
        "strict-transport-security": [
            {
                "key": "Strict-Transport-Security",
                "value": "max-age=31536000; includeSubdomains; preload"
            }
        ],
        "referrer-policy": [
            {
                "key": "Referrer-Policy",
                "value": "same-origin"
            }
        ],
        "x-xss-protection": [
            {
                "key": "X-XSS-Protection",
                "value": "1; mode=block"
            }
        ],
        "x-frame-options": [
            {
                "key": "X-Frame-Options",
                "value": "DENY"
            }
        ],
        "x-content-type-options": [
            {
                "key": "X-Content-Type-Options",
                "value": "nosniff"
            }
        ]
    }
}
```

When the same user logs in as `userB`, the browser will have all of `userB's` cookies, but also the access and refresh tokens of `userA`. When those cookies are sent back to CloudFront after the user has finished logging in as `userB`, the user will see a `Your request header section exceeds the maximum allowed size` error because with the additional cookies the maximum size of 8192 bytes is exceeded.

I'm signing out by navigating to `/signout`, my `cookieCompatibility` is set to `amplify` and these are my `cookieSettings`:
```json
"cookieSettings": {
    "idToken": null,
    "accessToken": null,
    "refreshToken": null,
    "nonce": null
}
```

I made the following change in the [sign-out handler](https://github.com/aws-samples/cloudfront-authorization-at-edge/blob/master/src/lambda-edge/sign-out/index.ts#L71) which prevents this.

```diff
const response = {
    status: "307",
    statusDescription: "Temporary Redirect",
    headers: {
      location: [
        {
          key: "location",
          value: `https://${
            CONFIG.cognitoAuthDomain
          }/logout?${stringifyQueryString(qs)}`,
        },
      ],
      "set-cookie": generateCookieHeaders.signOut({
        tokens: {
          id: cookies.idToken,
+         access: cookies.accessToken,
+         refresh: cookies.refreshToken,
        },
        ...CONFIG,
      }),
      ...CONFIG.cloudFrontHeaders,
    },
  };
```

Is it intentional behaviour that the access and refresh tokens are preserved when signing out?

**Comment by ottokruse** [comment]
Thanks for the report! Fixed this in #290

**Comment by ottokruse** [comment]
Fix released per [v2.3.1](https://github.com/aws-samples/cloudfront-authorization-at-edge/releases/tag/v2.3.1)
